tests/constants1.py

```diff
@@ -65,7 +65,9 @@ def test_str():
     s = str()
     s = str(5)
     s = str(-4)
-    s = str(5.6)
+    # TODO: This test was had hidden failure and was noticed during type
+    # checking.
```

Comment:
This test was not tested properly. There were no runtime functions matching the function signature. Introduced here: https://github.com/lcompilers/lpython/pull/110/commits/a1da515dac03aedade79d90a0c735a4e24181927